glue/plugins/export_plotly.py

```diff
@@ -305,6 +304,10 @@ def save_plotly(application, label):
     plotly.sign_in(user, apikey)
     plotly.plot(*args, **kwargs)
 
-exporters.add('Plotly', save_plotly, can_save_plotly, outmode='label')
-settings.add('PLOTLY_USER', 'Glue')
-settings.add('PLOTLY_APIKEY', 't24aweai14')
+def load_plugin():
+    from ..logger import logger
+    from ..config import exporters, settings
+    exporters.add('Plotly', save_plotly, can_save_plotly, outmode='label')
```

Comment:
Can you help me understand the distinction between adding an item to a registry, calling `add_plugin` on a registry, and calling `load_plugin`? 

Or a similar question, what is the difference between a plugin and an item in a registry?